tests/client/client_common.py

```diff
@@ -140,3 +142,27 @@ def test_build_and_run_instance(ebnt, container_name):
     time.sleep(.1)
 
     assert not instance.is_running()
+
+
+@pytest.mark.docker
+@pytest.mark.skipif(not has_docker(), reason='no docker installed')
+def test_build_and_delete_image(ebnt: Ebonite, model: Model):
```

Comment:
here you are testing client, but you checking the build of docker image. you should decouple testing of `ebnt.remove_image` and `DockerHost.remove_image` logic testing. For client testing I propose creating mock of `RuntimeEnvironment.Params` implementation (along with `Builder` and `Runner`) and just check that correct methods are called for those. And for actual image deletion testing you should test just it, for example pull the most lightweight image possible, check that it is there, then call `remove_image` and check if it is gone. No need to actually build `Model` image with server and all of this stuff - this is very time-consuming